docs/getting-started-guides/locally.md

```diff
@@ -17,55 +24,138 @@ At least [Docker](https://docs.docker.com/installation/#installation)
 ps`).  Some of the Kubernetes components need to run as root, which normally
 works fine with docker.
 
-#### etcd
-
-You need an [etcd](https://github.com/coreos/etcd/releases) in your path, please make sure it is installed and in your ``$PATH``.
-
-#### go
-
-You need [go](https://golang.org/doc/install) at least 1.3+ in your path, please make sure it is installed and in your ``$PATH``.
-
 ### Starting the cluster
```

Comment:
@dlsniper thank you.

Regarding comments - I mean that it would be great to leave comments, but as the plain text, not as a part of formatting (under the `code` formatting and with hash symbols `#`).

You don't need to delete the comments themselves - they are really needed for the users. But I'm asking to make them more readable for the webpage-style, not for the script-style.